tasks/owners.py

```diff
@@ -33,3 +36,37 @@ def guess_responsible(_, issue_id):
     owner = team_to_label(owner)
     print(owner)
     return owner
+
+
+def make_partition(names: list[str], owners_file: str, get_channels: bool = False) -> dict[str, set[str]]:
+    """
+    From a list of job / file names, will create a dictionary with the teams as keys and the names as values.
+
+    - If get_channels, the teams will be replaced by team channels.
+
+    Example
+    -------
+    If job1 belongs to team1 and team2, and job2 belongs to team2 and team3, the output will be:
+    {
+        "team1": {"job1"},
+        "team2": {"job1", "job2"},
+        "team3": {"job2"},
+    }
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Nice example!